specifications/api/swagger.yaml

```diff
@@ -760,11 +774,15 @@ x-stakePoolMetrics: &stakePoolMetrics
     non_myopic_member_rewards: *nonMyopicMemberRewards
     relative_stake:
       <<: *percentage
-      description: The pools stake compared to the stake of all pools.
+      description: |
+        The live pool stake relative to the *total* stake.
+
+        For more details, see the section "Relative Stake: Active vs Total" in
+        [Design Specification for Delegation and Incentives in Cardano](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/delegationDesignSpec/latest/download-by-type/doc-pdf/delegation_design_spec).
     saturation: *stakePoolSaturation
     produced_blocks:
       <<: *numberOfBlocks
-      description: Number of blocks produced by a given stake pool
+      description: Number of blocks produced by a given stake pool in its lifetime.
```

Comment:
> in its lifetime.

For added clarity. But this is not valid for jormungandr. Need to differentiate.

Reply:
Actually, it is already differentiated. This change is for the shelley one only.